statusHandler/termuiStatusHandler_test.go

```diff
@@ -0,0 +1,25 @@
+package statusHandler_test
```

Comment:
There are a lot of Exported Methods in the statusHandler. Please test all of them.